plugin_oar.py

```diff
@@ -64,6 +64,7 @@ def convert_qstat_to_yaml(self, orig_file, out_file, write_method):
         self.dump_all(out_file, self.stat_mapping[write_method])
 
 
+
```

Comment:
redundant line (there are quite a few of them, actually, in the source overall, better remove them)